app/frontend/src/features/messages/messagelist/MessageView.tsx

```diff
@@ -1,55 +1,123 @@
-import { Box, Card, CardContent, IconButton } from "@material-ui/core";
+import { Box, Card, CardContent, Typography } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
-import ExpandLessIcon from "@material-ui/icons/ExpandLess";
-import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
-import React, { useState } from "react";
+import React from "react";
 import Avatar from "../../../components/Avatar";
 import { Message } from "../../../pb/conversations_pb";
 import { timestamp2Date } from "../../../utils/date";
 import useCurrentUser from "../../userQueries/useCurrentUser";
 import { useUser } from "../../userQueries/useUsers";
 import TimeInterval from "./MomentIndication";
-import UserName from "./UserName";
+import classnames from "classnames";
 
-const useStyles = makeStyles({
-  root: { display: "flex" },
-  card: { flexGrow: 1 },
-  header: { display: "flex" },
-  name: { flexGrow: 1 },
-});
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    marginTop: theme.spacing(2),
+    "& > :first-child": { marginRight: theme.spacing(2) },
+  },
+  card: {
+    [theme.breakpoints.up("xs")]: {
+      width: "100%",
+    },
+    [theme.breakpoints.up("sm")]: {
+      width: "80%",
+    },
+    [theme.breakpoints.up("md")]: {
+      width: "70%",
+    },
+    border: "1px solid",
+    borderRadius: theme.shape.borderRadius,
+  },
+  userCard: {
+    borderColor: theme.palette.secondary.main,
+  },
+  otherCard: {
+    borderColor: theme.palette.primary.main,
+  },
+  header: {
+    display: "flex",
+    padding: theme.spacing(2),
+    paddingBottom: theme.spacing(1),
+    alignItems: "center",
+  },
+  footer: {
+    display: "flex",
+    paddingInline: theme.spacing(2),
+    paddingBottom: theme.spacing(2),
+    justifyContent: "flex-end",
+  },
+  messageTime: {
```

Comment:
Couldn't these styles live in `TimeInterval` instead, so it's closer to where it's needed?

Reply:
I think `TimeInterval` is just to abstract from moment, and might be used in all kinds of places?

Reply:
Though it's in messages. I'll change it.